scripts/utils/changelog.mjs

```diff
@@ -64,11 +64,11 @@ export function printEntries(entries) {
 
 export function replaceVersions(data, prevVer, newVer, isPatch = false) {
   return data
-    .replace(
+    .replaceAll(
       /prettier stable/gi,
       `Prettier ${isPatch ? prevVer : formatVersion(prevVer)}`
     )
-    .replace(
+    .replaceAll(
       /prettier main/gi,
```

Comment:
This looks a bit odd 🤔 (quite a few cases like that actually)

Reply:
Why?

Reply:
`replaceAll` + `/g`

Reply:
You don't have to know it's replacing all until you see the `g` flag.

https://github.com/sindresorhus/eslint-plugin-unicorn/issues/1948

Reply:
I did not know about the runtime error when using `replaceAll` without `/g` (https://github.com/ota-meshi/eslint-plugin-regexp/issues/472). Makes sense now!

Reply:
Approve?